modules/ROOT/pages/apicat-create-descriptor-file-cli.adoc

```diff
@@ -67,6 +67,36 @@ The presence and content of the designated descriptor file affects the result of
 
 * Existing file with content: The command fails and shows an error message that says the descriptor file already exists.
 
+=== Link Documentation Pages and Images through a Descriptor File
+
+You can add documentation pages and images to your APIs by adding a markdown file in the descriptor. You can visualize the image in the documentation page created for the Exchange Asset.
+
+Example descriptor file:
+
+----
+#%Catalog Descriptor 1.0
+projects:
+  - main: sources/api.raml
+    documentation:
+      - Summary: Summary.md
+      - Use Cases: Use_Cases.md
+    assetId: version-basic-api
+    apiVersion: v4
+    version: 4.0.0
+----
+With the `summary.md` file containing a link to an image (.png, .jpg) and a documentation markdown file:
```

Comment:
You can name the file "Summary.md" or "summary.md", but it should be consistent.